src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py

```diff
@@ -16,22 +16,23 @@
     STORAGE_RESOURCE_TYPE
 )
 from ._validators import (
-    validate_resource_group_name,
-    validate_admin_user
+    validate_resource_group_name
 )
 
 register_cli_argument('acr', 'registry_name',
                       options_list=('--name', '-n'),
                       help='Name of container registry',
                       completer=get_resource_name_completion_list(ACR_RESOURCE_TYPE))
 register_cli_argument('acr', 'storage_account_name',
-                      options_list=('--storage-account-name', '-s'),
                       help='Name of an existing storage account',
                       completer=get_resource_name_completion_list(STORAGE_RESOURCE_TYPE))
 
 register_cli_argument('acr', 'resource_group_name', resource_group_name_type)
 register_cli_argument('acr', 'location', location_type)
 register_cli_argument('acr', 'tags', tags_type)
+register_cli_argument('acr', 'admin_user_enabled',
+                      help='Whether the admin user account is enabled.',
+                      choices=['true', 'false'])
```

Comment:
This should just be a flag. See examples elsewhere in the CLI where we use `action='store_true'`.